src/main/java/core/basesyntax/db/Storage.java

```diff
@@ -0,0 +1,8 @@
+package core.basesyntax.db;
+
+import java.util.HashMap;
+import java.util.Map;
+
+public class Storage {
+    public static final Map<String, Integer> storage = new HashMap<String, Integer>();

```

Comment:
The `storage` field should be private to adhere to the encapsulation principle. Exposing it as public allows direct access to the underlying data structure, which can lead to uncontrolled modifications and violates the principle of encapsulation.

Reply:
Agree with AI mentor :) Let's make it private, also add methods to interact with it